### PR TITLE
cloud_functions: latestTokenData

### PR DESCRIPTION
**Pull request by panoel**

### CODE REVIEW

File: cloud_functions/src/getLatestTokenData.ts [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+const REFRESH_TIME_INTERVAL =
+  Number(process.env.CLOUD_FUNCTIONS_REFRESH_TIME_INTERVAL) || 1000 * 60 * 5;
+
+export async function getLatestTokenData(req: any, res: any) {
+  res.set('Access-Control-Allow-Origin', '*');
+  if (req.method === 'OPTIONS') {
+    // Send response to OPTIONS requests
+    res.set('Access-Control-Allow-Methods', 'GET');
+    res.set('Access-Control-Allow-Headers', 'Content-Type');
+    res.set('Access-Control-Max-Age', '3600');
```
Comment (barnjamin) [inline cloud_functions/src/getLatestTokenData.ts:57]: should we set this to a full day? not sure anything is paying attention to it but seems the max allowed is 86400

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Max-Age#directives

File: cloud_functions/src/getLatestTokenData.ts [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+      console.log('After getLatestTokenData =', timeDiff);
+      cache['values'] = values;
+      cache['lastUpdated'] = Date.now();
+    } else {
+      console.log(`cache is still valid, not refreshing ${new Date()}`);
+      values = cache['values'];
+    }
+    res.status(200).send(JSON.stringify(values));
+  } catch (e) {
+    res.sendStatus(500);
```
Comment (barnjamin) [inline cloud_functions/src/getLatestTokenData.ts:85]: log the error?

File: cloud_functions/src/getLatestTokenData.ts (hunk 1 above)
Comment (panoel) [reply cloud_functions/src/getLatestTokenData.ts:57]: This is the value set for most of the other cloud_functions.  Just being consistent. 
@evan-gray , what say you?

File: cloud_functions/src/getLatestTokenData.ts (hunk 2 above)
Comment (panoel) [reply cloud_functions/src/getLatestTokenData.ts:85]: Added logging.

**Review by evan-gray — DISMISSED** [review]

**Review by barnjamin — APPROVED** [review]

**Pull request merged**